library/src/layout/page.rs

```diff
@@ -87,6 +87,15 @@ pub struct PageElem {
     #[default(false)]
     pub flipped: bool,
 
+    ///When `{true}`, the left and right margins will be swapped on even pages
+    ///to achieve inside and outside margins. This is commonly used when writing
+    ///books, for example. Defaults to `{false}`.
+    /// ```example
+    /// #set page(two-sided: true, margin: (left: 1in, right: 1.25in))
+    /// ```
```

Comment:
```suggestion
    /// When `{true}`, the left and right margins will be swapped on even pages
    /// to achieve inside and outside margins. This is commonly used when writing
    /// books, for example. Defaults to `{false}`.
    ///
    /// ```example
    /// #set page(two-sided: true, margin: (left: 1in, right: 1.25in))
    /// ```
```

Reply:
I added the leading spaces and re-wrapped the lines to be at maximum 80 characters.

Reply:
Thanks!